rbpf-cli/src/main.rs

```diff
@@ -0,0 +1,226 @@
+use clap::{App, Arg};
+use serde::{Deserialize, Serialize};
+use serde_json::Result;
+use solana_bpf_loader_program::{
+    BpfError,
+    create_vm,
+    serialization::serialize_parameters,
+    syscalls::register_syscalls,
+    ThisInstructionMeter,
+};
+use solana_rbpf::{
+    assembler::assemble,
+    static_analysis::Analysis,
+    verifier::check,
+    vm::{
+        Config, DynamicAnalysis, Executable,
+    },
+};
+use solana_sdk::{
+    account::AccountSharedData,
+    bpf_loader,
+    keyed_account::KeyedAccount,
+    process_instruction::{InvokeContext, MockInvokeContext},
+    pubkey::Pubkey,
+};
+use std::{cell::RefCell, fs::File, io::Read, path::Path};
+
+#[derive(Serialize, Deserialize)]
+struct Data {
+    accounts: Vec<AccountSharedData>,
+    insndata: Vec<u8>,
+}
+fn load_accounts<'a>(path: &Path) -> Result<Data> {
+    let file = File::open(path).unwrap();
+    let input: Data = serde_json::from_reader(file)?;
+    println!("accounts {:?}", &input.accounts);
+    println!("insndata {:?}", &input.insndata);
+    Ok(input)
+}
+
+fn main() {
+    solana_logger::setup();
+    let matches = App::new("Solana BPF CLI")
+        .version("0.2.8")
+        .author("Solana Maintainers <maintainers@solana.foundation>")
+        .about("CLI to test and analyze eBPF programs")
+        .arg(
+            Arg::new("assembler")
```

Comment:
Can we auto-detect this by looking for elf magic?

Reply:
Sounds like a good idea.